nuxeo-user-group-management/nuxeo-create-group.html

```diff
@@ -132,11 +132,12 @@
 
     <form is="iron-form" id="form" class="vertical layout">
 
-      <paper-input id="groupName" label="[[i18n('createGroup.group.name')]]" value="{{groupName}}" required always-float-label></paper-input>
-      <paper-input id="groupLabel" label="[[i18n('createGroup.group.label')]]" value="{{groupLabel}}" always-float-label></paper-input>
+      <paper-input id="groupName" label="[[i18n('createGroup.group.name')]]" name="groupname" value="{{groupName}}" required always-float-label></paper-input>
+      <paper-input id="groupLabel" label="[[i18n('createGroup.group.label')]]" name="grouplabel" value="{{groupLabel}}" always-float-label></paper-input>
```

Comment:
no need for these name attributes since we already have unique ids for these elements